development_scripts/demo_ecms_calibration_curve.py

```diff
@@ -48,16 +48,41 @@
 
 plugins.activate_siq()
 
-cal = ecms.ecms_calibration_curve(
+# The following issues a warning and returns an ixdat MSCalResult :)
+cal_1 = ecms.ecms_calibration_curve(
     mol="H2",
     mass="M2",
     n_el=-2,
     tspan_list=[[600, 700], [1150, 1250], [1800, 1900], [2350, 2450]],
 )
-siq_cal = ecms.siq_ecms_calibration_curve(
+# The following returns a siq CalPoint :)
+siq_cal_2 = ecms.siq_ecms_calibration_curve(
     mol="H2",
     mass="M2",
     n_el=-2,
     tspan_list=[[600, 700], [1150, 1250], [1800, 1900], [2350, 2450]],
+    tspan_bg=[400, 450],
+    force_through_zero=True,
 )
 
+# And here we show all the necessary interconversions:
+
+cal_2 = MSCalResult.from_siq(siq_cal_2)
+print(cal_2)
+
+siq_cal_1 = cal_1.to_siq()
+print(siq_cal_1)
+
+# You can't directly add MSCalResults, but that operation *is* implemented in siq :)
+siq_calibration = cal_1.to_siq() + cal_2.to_siq()
+print(siq_calibration)
+
+# The following dowsn't work because it's a SensitivityList instead of a Calibration :(
+# siq_calibratoin.plot_as_spectrum()
```

Comment:
# siq_calibration.plot_as_spectrum() (spelling)

Reply:
Thanks for the typo catch! 
Yeah, of course, one would never actually do this. It was just to show that all of the interconversions work!